frappe/utils/safe_exec.py

```diff
@@ -181,6 +181,7 @@ def get_safe_globals():
 		run_script=run_script,
 		is_job_queued=is_job_queued,
 		get_visible_columns=get_visible_columns,
+		lang=getattr(frappe.local, "lang", "en"),
```

Comment:
Better to add this as `frappe.lang` instead to avoid potentially breaking things?

Also frappe.lang is usable outside of server scripts too so it's consistent behaviour.